docs/install.md

```diff
@@ -36,28 +50,34 @@ Your Sugarizer server will start automatically and will be accessible on http://
 To run Sugarizer Server **without Docker**, follow the step behind. Commands are shown from a new Debian Linux machine and could be different for other Linux distribution or for an already installed machine:
 
 
-**Install Node.js**: Install Node.js (6+) and npm to manage packages. See [here](http://nodejs.org/ "here") more information.
+**Install Node.js**: Install Node.js (6+) and npm to manage packages. See [here](http://nodejs.org/ "here") for more information.
 
-    sudo apt-get install nodejs
+    sudo apt update
+	sudo apt install nodejs
 
-**Install MongoDB**: Install MongoDB (2.6+). Don't forget to create a /data/db directory to store databases. See [here](http://www.mongodb.org/ "here") more information.
+**Install MongoDB**: Install MongoDB (2.6+). Don't forget to create a /data/db directory to store databases. See [here](https://docs.mongodb.com/manual/installation/ "here") for more information.
 
     sudo apt-get install mongodb
     sudo mkdir -p /data/db
 
+**Install Git**: Install Git (2.17+). See [here](https://github.com/git-guides/install-git) for more information.
```

Comment:
It's fine to assume that the user have Git installed. Also, Git is not something which is required to run Sugarizer Server. So this part is not needed.